### DISCUSSION TITLE
Is it possible to customize how selected value display?

### DISCUSSION THREAD

**Discussion by Toshinaki** [mantinedev/mantine#1426] [Q&A] [answered]
```
<Select
  placeholder="Mode"  
  data={[
    { value: '1', label: '1' },
    { value: '2', label: '2' },
  ]}
/>
```
For this select, after select option 2, I'd like the Select displays "Mode: 2" instead of just "2".

We could [customize options](https://mantine.dev/core/select/#custom-item-component) but I didn't find any way to customize selected value.

Is this possible? If not, can we have this feature maybe in a later version?

**Comment by rtivital** [comment, marked as answer]
No, it is not possible, Select component is a regular input, we also do not plan to add support for custom value renderer any time soon

**Reply by Toshinaki** [reply]
Ok thanks.
Why not BTW? I can see that `MultiSelect` and `Select` share similar implementations, and `MultiSelect` has a `valueComponent` to render values.
This [example](https://mantine.dev/core/select/#custom-item-component) from document shows a `Select` with custom item component. And it's kind of strange to have a pulldown with rich information but only text showing the selection.
